esmvaltool/diag_scripts/arctic_ocean/utils.py

```diff
@@ -220,15 +220,19 @@ def get_cmap(cmap_name):
     Additional custom colormap for salinity is provided:
     - "custom_salinity1"
     """
-    cm.register_cmap(cmap=LinearSegmentedColormap(
-        'cubehelix3', mpl._cm.cubehelix(gamma=1.0, s=2.0, r=1.0, h=3)))
+    try:
+        mpl.colormaps.register(cmap=LinearSegmentedColormap(
+            'cubehelix3', mpl._cm.cubehelix(gamma=1.0, s=2.0, r=1.0, h=3)),
+                               name="new_cubehelix3", force=False)
+    except:
```

Comment:
could you please use the except + error type here? eg `except ValueError:` or whatever the actual raised exception type is